views/Main.react.js

```diff
@@ -574,8 +574,20 @@ class MainPage extends Component {
     default_acl = options;
     this.starAcl(options, this.state.star)
     this.setState({
-      totalStudies: acl.length,
-      totalSubjects: options.length,
+      // study determination: at least one upper case
+      totalStudies: acl.filter(s=> /[A-Z]/.test(s.study)).length,
+      // subject determination: at least 2 alpha and 3 numeric
```

Comment:
what does this mean? A subject is a subject with 2 alpha characters and 3 numeric characters?